### PR TITLE
install protobuf compiler from releases instead of apt

### PR DESCRIPTION
**Pull request by cbeck88**
Hi, would you take a PR to uprev protobuf?

This gets to 1.25, from 1.12 which is 4 years old

Thanks for your work

### CODE REVIEW

**Comment by cbeck88** [comment]
For comparison, this is what is installed by `apt install protobuf-compiler` in jammy: https://packages.ubuntu.com/jammy/i386/protobuf-compiler/filelist

File: Dockerfile [hunk 1]
```diff
@@ -30,11 +30,18 @@ RUN apt-get update && apt-get install -y \
...
   libtool \
-  protobuf-compiler \
   libprotobuf-dev \
+  unzip \
   --no-install-recommends && \
   rm -rf /var/lib/apt/lists/*
 
+# Install a more recent release of protoc (protobuf-compiler in jammy is 4 years old and misses some features)
+RUN cd /tmp && \
+    curl -sSL https://github.com/protocolbuffers/protobuf/releases/download/v25.2/protoc-25.2-linux-x86_64.zip -o protoc.zip && \
```
Comment (clux) [inline Dockerfile:40]: could we extract the version into the evars below?

```suggestion
    curl -sSL https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VER}/protoc-${PROTOC_VER}-linux-x86_64.zip -o protoc.zip && \
```

**Review by clux — COMMENTED** [review]
thanks a lot, looks sensible to me. minor request on top of it

**Comment by clux** [comment]
Thanks!

**Pull request merged**